tests/unit/amo/components/TestApp.js

```diff
@@ -22,12 +22,18 @@ import {
   createContextWithFakeRouter,
   dispatchClientMetadata,
   dispatchSignInActions,
-  fakeCookie,
+  fakeCookies,
   fakeI18n,
   shallowUntilTarget,
   userAuthToken,
 } from 'tests/unit/helpers';
 
+// Skip `withCookies` HOC since Enzyme does not support the React Context API.
+// See: https://github.com/mozilla/addons-frontend/issues/6839
+jest.mock('react-cookie', () => ({
```

Comment:
Can you just pull this into a helper like `setUpMockReactCookie()` ? This will make it easier to find and delete all the code when we remove it. I don't know what's up with the `jest` magic global so if that doesn't work then it's fine.

Reply:
Looks like it does not work well...